docs/content/guides/developer/first-app/publish.mdx

```diff
@@ -9,7 +9,7 @@ To publish your package to the Sui network, use the `publish` CLI command in the
 {@include: ../../../snippets/info-gas-budget.mdx}
 
 ```sh
-$ sui client publish --gas-budget 5000000
+$ sui client publish
```

Comment:
What's missing in this view is that include statement points out that gas-budget is no longer needed. I left the mentions of the option in the command throughout docs. I wonder if it's time to remove those includes as well.